src/internal/observable/race.ts

```diff
@@ -31,7 +31,18 @@ export function race<T>(...observables: ObservableInput<T>[]): Observable<T>;
 export function race(...observables: ObservableInput<any>[]): Observable<{}>;
 
 /**
- * Returns an Observable that mirrors the first source Observable to emit an item.
+ * Returns an observable that mirrors the first source observable to emit an item.
+ *
+ * ![](race.png)
+ *
+ * `race` returns an observable, that when subscribed to, subscribes to all source observables immediately.
+ * As soon as one of the source observables emits a value, the result unsubscribes from the other sources.
+ * The resulting observable will forward all emissions, including error and completion, from the "winning"
```

Comment:
I would suggest using the wording notification instead of emission, which is more close to the internal naming conventions.

Reply:
Thanks for suggession. I have made the changes as suggested.